com.basistech.m2e.code.quality.checkstyle.test/META-INF/MANIFEST.MF

```diff
@@ -7,4 +7,5 @@ Bundle-Vendor: Basis Technology Corp.
 Fragment-Host: com.basistech.m2e.code.quality.checkstyle;bundle-version="[1.1.0,2.0.0)"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Require-Bundle: org.junit,
+ com.basistech.m2e.code.quality.shared.test;bundle-version="1.1.0",
```

Comment:
Perhaps the Version number here (and elsewhere like it?) can just be dropped?

```suggestion
 com.basistech.m2e.code.quality.shared.test,
```

Reply:
changed as requested in all 4 places